ring-core/src/ring/util/response.clj

```diff
@@ -92,13 +92,22 @@
       (.exists file)
         file)))
 
-(defn- file-content-length [resp]
-  (let [file ^File (:body resp)]
-    (header resp "Content-Length" (.length file))))
+(defn- file-info [^File file]
+  (if file
```

Comment:
Because there isn't a nil check on line 120, but it's probably better to have the caller check. Will fix.